apps/angular/src/shared/components/movie-type/movie-type.component.ts

```diff
@@ -0,0 +1,41 @@
+import { ChangeDetectionStrategy, Component, HostBinding, Input } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { AnimeType } from '@js-camp/core/models/anime-type';
+
+const ANIME_TYPE_COLORS: Record<AnimeType, string> = {
+	[AnimeType.Movie]: 'anime-type_movie',
+	[AnimeType.Music]: 'anime-type_music',
+	[AnimeType.ONA]: 'anime-type_ona',
+	[AnimeType.OVA]: 'anime-type_ova',
+	[AnimeType.PromotionalVideos]: 'anime-type_promotional-videos',
+	[AnimeType.Special]: 'anime-type_special',
+	[AnimeType.TV]: 'anime-type_tv',
+	[AnimeType.Unknown]: 'anime-type_unknown',
+};
+
+/** Anime type. */
+@Component({
+	selector: 'camp-movie-type',
```

Comment:
```suggestion
	selector: 'camp-anime-type',
```
Check the entire file